incubator/docker-registry/templates/registry.yaml

```diff
@@ -0,0 +1,89 @@
+{{- $replicas := .Values.replicas }}
```

Comment:
I guess you misunderstood me. It's ok to have the stuff configurable. I just don't see the need to create a local variable here for every value.